trim.C

```diff
@@ -355,6 +355,7 @@ TrimBase::trim(IonBase * pka, std::queue<IonBase*> & recoils)
           vacancyCreation();
         } else {
           // nope, the _pka gets stuck at that site as...
+          replacementCollision();
```

Comment:
replacement in this sense now includes replacement & substitution. I guess that is what we want.